app/services/analytics_events.rb

```diff
@@ -521,6 +521,24 @@ def idv_native_camera_forced(
     )
   end
 
+
+  # @param [Boolean] use_newer_sdk Whether not to load the newer Acuant SDK version
+  # @param [String] version The version of the Acuant SDK that was loaded
+  # An A/B test for loading the old vs new version of the Acuant SDK.
+  # If this analytics call is not made at all, then the test isn't enabled.
+  def idv_acuant_sdk_upgrade_a_b_test(
```

Comment:
based on the wording of the JIRA, I think that it might be good to add AB test attributes to the existing `idv_doc_auth_submitted_image_upload_vendor` or something, so that we could easily correlate A/B test bucket with results

having a separate event like this to proactively log which bucket is good too in case users don't make it as far as submissions

Reply:
Yeah we need to know at SDK load time (ie before any submission or even attempts). Should we do both, keep the new event and amend the upload vendor event?

Reply:
yeah seems like it would be useful to have the info at both events

Reply:
The way that `flow_path` is tracked across events during document capture could serve as good prior art to follow:

https://github.com/18F/identity-idp/blob/9657a8371cfbdd4d03f342dc4c438b21a94c0833/app/javascript/packs/document-capture.tsx#L69-L72